src/Hosting/Hosting/src/IActivityFeature.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.AspNetCore.Hosting
+{
+    /// <summary>
+    /// Feature to access the <see cref="Activity"/> associated with a request
+    /// </summary>
+    public interface IActivityFeature
```

Comment:
It should be there. @shirhatti is just new here 😄 . Also it should be called IHttpActivityFeature

Reply:
Way to drag me on here 😄 

I'll have to add a reference to System.Diagnostics.DiagnosticSource in Http.Features. Is that okay?

Reply:
Woops, that should be here: https://github.com/dotnet/aspnetcore/tree/main/src/Http/Http.Features/src

As a nuget package reference? That's a legit concern since this is one of our more broadly targeted packages.

Reply:
Ifdef

Reply:
I think this is resolved